tests/purs/passing/VisibleTypeApplications.purs

```diff
@@ -78,4 +78,16 @@ leftSkip = Left @_ @String 0
 rightSkip :: Either Int String
 rightSkip = Right @Int @_ "0"
 
+hrk :: forall (@f :: forall k. k -> Type). Proxy f
+hrk = Proxy
+
+hrk' :: Proxy Proxy
+hrk' = hrk @Proxy
```

Comment:
I mentioned this in the PureScript Discord, but this is unlike doing:
```hs
hrk' :: Proxy Proxy
hrk' = hrk
```
Which fails with the following error:
```hs
  Could not match kind
  
    forall k. k -> Type
  
  with kind
  
    forall k. k -> Type
```
I suspect this is because of impredicativity of some sort.